### ISSUE TITLE
Slackware

### ISSUE THREAD

**Issue by CountZachula** [egee-irl/distro-delves#97]
![Slackware Logo](http://www.slackware.com/grfx/shared/slackware_traditional_website_logo.png)

## Name & tl;dr

- Name: Slackware
- Version: 14.2
- TL;DR: The oldest distribution that is still maintained. Designed for stability and to be the most "UNIX-like" Linux distribution. Flexible enough for home and server use. [Parent to many distributions](https://en.wikipedia.org/wiki/Slackware#/media/File:SlackwareFamilyTree1210.svg), most notably SUSE. Fantastic system for learning Linux concepts, albeit a bit daunting at first. Maintained by Patrick Volkerding, Slackware's [Benevolent Dictator for Life](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life).

## What is the release date or cycle?
**Rolling** since 14.2's release in 2016. Fixed every year prior, going all the way back to 1993.

## Who is the target user?

- Intermediate to advanced home users who enjoy modern applications and maintaining traditional Linux metaphors.
- IT professionals looking for stability, no complex enterprise agreements,

## What platforms does it support?

IA-32, x86-64, ARM

## What makes it special?

- DIY distro
- Nostalgia
- Coming from the BSD world? Overwhelmed with the number of distros and upstream packages being modified? Slackware will feel very familiar to you, as one of its core principles is keeping it as *nix as possible.
- If you know Slackware, you really know Linux.

## Other
Slackware holds a very place in my heart. It was my first exposure to Linux and fostered the technical foundation I have today, personally and professionally.

- http://www.slackware.com/
- https://docs.slackware.com/
- https://slackbuilds.org/

**Comment by egee-irl** [comment]
I may have asked this before but does this have a cli installer or is it dyi like Arch?

I could see making a special event for something like this. Probably not a regular Saturday episode though.

**Comment by CountZachula** [comment]
@egee-irl It uses a text-based, menu-driven installer for a majority of the components, including a desktop environment. I find it easier than Arch to jump in. You'll spend more time in the installer than the shell, **but** there are some manual steps like creating a user, configuring your system to boot into X by default, etc.

That said, maybe it's too much for a Saturday episode. Up to you. Maybe a special event for Slackware's birthday on July 16th next year?

**Comment by egee-irl** [comment]
We might look at this tomorrow :)
